### PR TITLE
[16.0][ADD] base_wamas_ubl: Add menu entries to help debug

### PR DESCRIPTION
**Pull request by tuantrantg**

### CODE REVIEW

File: base_wamas_ubl/wizard/check_wamas_file_wizard.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+import json
+import struct
+from base64 import b64decode
+
+from odoo import _, api, fields, models
+
+
+class CheckWamasFileWizard(models.TransientModel):
+    _name = "check.wamas.file.wizard"
```
Comment (simahawk) [inline base_wamas_ubl/wizard/check_wamas_file_wizard.py]:
```suggestion
    _name = "wamas.ubl.wiz.check"
```

File: base_wamas_ubl/wizard/simulate_wamas_file_wizard.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+# Copyright 2023 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from base64 import b64decode, b64encode
+
+from odoo import _, api, fields, models
+
+
+class SimulateWamasFileWizard(models.TransientModel):
+    _name = "simulate.wamas.file.wizard"
```
Comment (simahawk) [inline base_wamas_ubl/wizard/simulate_wamas_file_wizard.py]:
```suggestion
    _name = "wamas.ubl.wiz.simulate"
```

File: base_wamas_ubl/wizard/simulate_wamas_file_wizard.xml [hunk 3]
```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+
+    <record id="simulate_wamas_file_wizard_form" model="ir.ui.view">
+        <field name="name">simulate.wamas.file.wizard.form</field>
+        <field name="model">simulate.wamas.file.wizard</field>
+        <field name="arch" type="xml">
+            <form string="Simulate WAMAS File">
+                <group>
+                    <field name="wamas_file" filename="wamas_filename" />
```
Comment (simahawk) [inline base_wamas_ubl/wizard/simulate_wamas_file_wizard.xml]: an explanation of what both wizards do in an info box would be nice to have

**Review by simahawk — COMMENTED** [review]
LG overall, minor remarks

**Comment by tuantrantg** [comment]
hi @simahawk the PR's updated as change requests, please review it again, thank you

**Review by simahawk — APPROVED** [review]

**Comment by simahawk** [comment]
@tuantrantg can you rebase pls

**Comment by tuantrantg** [comment]
hi @simahawk the PR's rebased, please check it again, thank you

**Pull request merged**